### PR TITLE
Better seed

### PR DESCRIPTION
**Pull request by itsOrD**

### CODE REVIEW

File: scratch.txt [hunk 1]
```diff
@@ -31,4 +31,21 @@
   const Link = styled.a`
     text-decoration: none;
     font-size: 8px;
-  `;
\ No newline at end of file
+  `;
+
+
```
Comment (kmorgan8588) [inline scratch.txt:36]: helpful notes for later?

File: package.json [hunk 2]
```diff
@@ -12,9 +12,11 @@
...
-    "build": "webpack -d --watch",
-    "server": "nodemon server/index.js",
-    "seed": "node server/index.js",
+    "build": "weback -d",
+    "build-dev:": "webpack -d --watch",
+    "start": "nodemon server/index.js",
+    "server": "node server",
+    "seed": "node db/seedScript.js",
     "schema": "mysql -uroot < db/schema.sql"
   },
```
Comment (kmorgan8588) [inline package.json:21]: you should probably make use of the helpful start script, it's helpful :)

**Review by kmorgan8588 — APPROVED** [review]
good changes, glad the dockerizing worked out in the end

File: scratch.txt (hunk 1 above)
Comment (itsOrD) [reply scratch.txt:36]: meant to keep that in .gitignore; will fix

File: package.json (hunk 2 above)
Comment (itsOrD) [reply package.json:21]: Good point, will swap that.

**Comment by itsOrD** [comment]
Going to do a new PR with these minor changes instead of merging this one.

**Pull request closed without merging**